spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1,9 +1,45 @@
 [[kubernetes-getting-started]]
 = Getting Started
 
-== Deploying Streams on Kubernetes
+[[dataflow-kubernetes-intro]]
+== Introducing Spring Cloud Data Flow for Kubernetes
+
+Spring Cloud Data Flow is a toolkit for building data integration and real-time data processing pipelines.
+
+Pipelines consist of Spring Boot apps, built using the Spring Cloud Stream or Spring Cloud Task microservice frameworks. This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to event streaming and predictive analytics.
+
+This project provides support for using Spring Cloud Data Flow with Kubernetes as the runtime for these pipelines with apps packaged as Docker images.
+
+[[spring-cloud-data-flow-overview]]
+=== Spring Cloud Data Flow
+Spring Cloud Data Flow is a cloud-native orchestration service for composable data microservices on modern runtimes. With Spring Cloud Data Flow, developers can create and orchestrate data pipelines for common use cases such as data ingest, real-time analytics, and data import/export.
```

Comment:
I'd suggest removing the overview sections. We removed it from the core docs, too. Besides, there are 2 different types of core message discussed here. The intro and then this section, that is.